core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-edit-content-category-field-dialog/dot-edit-content-category-field-dialog.component.scss

```diff
@@ -0,0 +1,37 @@
+@use "variables" as *;
+
+:host {
+    display: block;
+    height: 100%;
+
+    .category-field__dialog {
+        display: grid;
+        grid-template-columns: 2fr 1fr;
+        max-width: 1200px;
```

Comment:
@oidacra Should this field be a variable?

Reply:
I will leave it as is, because, we need to convert the dialog to a sidebar.